partcad-cli/src/partcad_cli/click/commands/list/assemblies.py

```
@@ -0,0 +1,76 @@
+            for assy_name, assy in project.assemblies.items():
+                if used_by is not None and assy.count == 0:
+                    continue
+
+                line = "\t"
+                if recursive:
+                    line += f"{project_name}"
+                    line += " " + " " * (35 - len(project_name))
+                line += f"{assy_name}"
+                if used_by is not None:
+                    assy = project.assemblies[assy_name]
+                    line += "(%d)" % assy.count
```

Ruff (0.8.2)

Use format specifiers instead of percent format

(UP031)